locations.pl

```diff
@@ -77,6 +54,35 @@
 describe_location(forest_hut) :-
 	write("You're standing in front of a small forest hut but the door is locked."), nl.
 	
+describe_items(Location) :-
+	position(Item, Location),
+	is_item(Item),
+	write("There is "), describe_item(Item),
+	write("Write 'take.' to take this item."), nl.
+
+describe_items(_).
+
+describe_directions :-
```

Comment:
It was a little bit annoying while testing game when I only wanted to reach the place as soon as possible and didn't get it right.
Personally I will add it to the game, maybe after invoking look_around command we will desribe it